capstone-project/src/main/angular-webapp/src/styles.css

```diff
@@ -0,0 +1,12 @@
+/* You can add global styles to this file, and also import other style files */
+#map-container {
```

Comment:
Why not define the map container's style in the map.component.css file? I think it will fit better in that file since this element appears in the map.component.html file and not in the index.html file.